mlrun/api/schemas/model_endpoints.py

```diff
@@ -183,3 +297,40 @@ class GrafanaTimeSeriesTarget(BaseModel):
 
     def add_data_point(self, data_point: GrafanaDataPoint):
         self.datapoints.append((data_point.value, data_point.timestamp))
+
+
+def _mapping_attributes(
+    base_model: BaseModel,
+    flattened_dictionary: dict,
+    json_parse_values: typing.List = None,
+):
```

Comment:
This method is the "from_dict" of a `ModelObj` no?

Reply:
no, it's a new function that I've defined for generating a `BaseModel` object from a provided dictionary